docs/IntelOwl/contribute.md

```diff
@@ -424,6 +424,107 @@ You can use the django management command `dumpplugin` to automatically create t
 
 Example: `docker exec -ti intelowl_uwsgi python3 manage.py dumpplugin PlaybookConfig <new_analyzer_name>`
 
+## DataModel
+In version XXX of IntelOwl, a new plugin has been added: the `DataModel`.
```

Comment:
As you can see in the updated PR template, from now on we add a new line at the start of every new documentation section to document the version of the new feature: https://github.com/intelowlproject/docs/blob/main/.github/pull_request_template.md